src/App.js

```diff
@@ -8,6 +7,7 @@ import Footer from './components/Footer/Footer';
 import axios from 'axios';
 import { HttpDopplerLegacyClient } from './services/doppler-legacy-client';
 import { OnlineSessionManager } from './services/session-manager';
+import Reports from './components/Reports/Reports';
```

Comment:
Why there is a reports inside reports here? What's the logic here?

Reply:
Reports it's the folder contain all reports components. And the other reports it's the Reports page